tests/library-linkstatic-flag/BUILD.bazel

```diff
@@ -94,11 +94,22 @@ sh_inline_test(
     script = """
 set -euo pipefail
 is_dynamic () {
-    # taken from https://github.com/NixOS/nixpkgs/blob/0b3f50f844e2a6b507b18d7c5259bb850b382f87/pkgs/build-support/setup-hooks/auto-patchelf.sh#L167-L170
-    readelf -l -- "$1" | grep -q "^ *INTERP\\>"
+    local res=1
+    if type file >/dev/null; then
+        file --dereference -- "$1"
+        file --dereference -- "$1" | grep -q "dynamically linked"
```

Comment:
Why the double `file --dereference` instead of `set -o pipefail`?

Reply:
Makes it easier to debug if it should fail again in the future (it’s just a test, so the output doesn’t matter for anything else)